proselint/command_line.py

```diff
@@ -37,20 +37,24 @@ def timing_test(corpus="0.1.0"):
     for file in os.listdir(corpus_path):
         filepath = os.path.join(corpus_path, file)
         if ".md" == filepath[-3:]:
-            subprocess.call(
-                "proselint {} >/dev/null".format(filepath), shell=True)
+            subprocess.call(["proselint", filepath, ">/dev/null"])
```

Comment:
This doesn't really work. Without using `shell=True`, the `>` is just another character, so you're just passing an extra file name. You can redirect using something like `stdout=subprocess.DEVNULL`.